rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewTreeObserverGlobalLayoutEventOnSubscribe.java

```diff
@@ -0,0 +1,52 @@
+package com.jakewharton.rxbinding.view;
+
+import android.os.Build;
+import android.view.View;
+import android.view.ViewTreeObserver;
+import com.jakewharton.rxbinding.internal.MainThreadSubscription;
+import rx.Observable;
+import rx.Subscriber;
+
+import static com.jakewharton.rxbinding.internal.Preconditions.checkUiThread;
+
+final class ViewTreeObserverGlobalLayoutEventOnSubscribe
+    implements Observable.OnSubscribe<ViewTreeObserverGlobalLayoutEvent> {
+  private final View view;
+
+  public ViewTreeObserverGlobalLayoutEventOnSubscribe(View view) {
+    this.view = view;
+  }
+
+  @Override public void call(final Subscriber<? super ViewTreeObserverGlobalLayoutEvent> subscriber) {
+    checkUiThread();
+
+    final ViewTreeObserver.OnGlobalLayoutListener listener = new ViewTreeObserver.OnGlobalLayoutListener() {
+      @Override public void onGlobalLayout() {
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.JELLY_BEAN) {
+          view.getViewTreeObserver().removeOnGlobalLayoutListener(this);
```

Comment:
It's a one-shot triggering for typical use cases (e.g., getting the measured width & height).